aemcmc/conjugates.py

```diff
@@ -268,13 +268,99 @@ def local_beta_negative_binomial_posterior(fgraph, node):
     return rv_var.owner.outputs
 
 
+def uniform_pareto_conjugateo(observed_val, observed_rv_expr, posterior_expr):
+    r"""Produce a goal that represents the application of Bayes theorem
+    for a pareto prior with a uniform with 0 as the lower bound observation model.
+
+    .. math::
+        Y \sim \operatorname{Uniform}\left(0, \theta\right)
+
+
+
+    Parameters
+    ----------
+    observed_val
+        The observed value.
+    observed_rv_expr
+        An expression that represents the observed variable.
+    posterior_exp
+        An expression that represents the posterior distribution of the latent
+        variable.
+
+    """
+    # beta-negative_binomial observation model
+    x_lv, k_lv = var(), var()
+    theta_rng_lv = var()
+    theta_size_lv = var()
+    theta_type_idx_lv = var()
+    theta_et = etuple(
+        etuplize(at.random.pareto),
+        theta_rng_lv,
+        theta_size_lv,
+        theta_type_idx_lv,
+        k_lv,
+        x_lv,
+    )
+    Y_et = etuple(etuplize(at.random.uniform), var(), var(), var(), 1, theta_et)
+
+    # new_x_et = at.max(observed_val)
```

Comment:
I am having most trouble right here, where I am not sure how to get the at.max(obs, x_m)